packages/@tailwindcss-vite/src/index.ts

```diff
@@ -169,7 +168,7 @@ export default function tailwindcss(): Plugin[] {
       },
       transform(src, id, options) {
         let extension = getExtension(id)
-        if (extension === '' || extension === 'css') return
```

Comment:
This is an optimization that makes sure vite `<style>` blocks aren't scanned for candidates in addition to `css` files.